packages/app-builder/src/components/Decisions/Filters/FilterDetail/ScheduledExecutionFilter.tsx

```diff
@@ -0,0 +1,124 @@
+import { Highlight } from '@app-builder/components/Highlight';
+import { type ScheduledExecutionsLoader } from '@app-builder/routes/ressources+/decisions+/list-scheduled-execution';
+import { formatDateTime, useFormatLanguage } from '@app-builder/utils/format';
+import { getRoute } from '@app-builder/utils/routes';
+import { useFetcher } from '@remix-run/react';
+import { matchSorter } from 'match-sorter';
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { useSpinDelay } from 'spin-delay';
+import { Input, SelectWithCombobox } from 'ui-design-system';
+
+import { decisionsI18n } from '../../decisions-i18n';
+import { useScheduledExecutionFilter } from '../DecisionFiltersContext';
+
+export function ScheduledExecutionFilter() {
+  const { t } = useTranslation(decisionsI18n);
+  const language = useFormatLanguage();
+
+  const loadFetcher = useFetcher<ScheduledExecutionsLoader>();
+  React.useEffect(() => {
+    if (loadFetcher.state === 'idle' && !loadFetcher.data) {
+      loadFetcher.load(
+        getRoute('/ressources/decisions/list-scheduled-execution'),
+      );
+    }
+  }, [loadFetcher]);
+  const successfullScheduledExecutions = React.useMemo(
+    () =>
+      loadFetcher.data
+        ?.filter(({ status }) => status === 'success')
```

Comment:
This is valid today but is probably no longer going to be pretty soon, in fact as soon as I start making scheduled execs execute piece by piece instead of in one big transaction.
So: works right now, but we might as well not have this logic and wording implemented to begin with.

Reply:
If there is still a valid condition to filter all scheduled executions we want to display, it's ok